packages/storybook/cxl-ui/cxl-credential/default.stories.js

```diff
@@ -0,0 +1,70 @@
+import { html } from 'lit';
+import '@conversionxl/cxl-ui/src/components/cxl-credential.js';
+
+export default {
+  title: 'CXL UI/cxl-credential',
+};
+
+const Template = ({ credentialId, fullName, title, type, date, credentialUrl }) => html`
+  <section style="display: flex; flex-direction: column; gap: 32px; align-items: center">
+    <cxl-credential
+      credential-id=${credentialId}
+      full-name=${fullName}
+      title=${title}
+      type=${type}
+      date=${date}
+    ></cxl-credential>
+
+    <cxl-credential
+      theme="dark"
+      credential-id=${credentialId}
+      full-name=${fullName}
+      title=${title}
+      type=${type}
+      date=${date}
+      credential-url=${credentialUrl}
+    ></cxl-credential>
+  </section>
+`;
+
+export const CourseShort = Template.bind({});
+export const CourseLongTitle = Template.bind({});
+export const CourseLongName = Template.bind({});
+
+export const MinidegreeShort = Template.bind({});
```

Comment:
Instead of separate stories for minidegree, could we use a select control?

Reply:
This change is still pending, will get to it tomorrow